build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
       'compileSdk': 27,
       'buildTools': '27.0.3',
 
-      'supportLibrary': '27.0.2',
+      'supportLibrary': '25.3.1',
```

Comment:
cant we just bump the minSDK to 14 here and align it with suppLib? @JakeWharton I guess its time no? :D

Reply:
Yes.